Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java

```diff
@@ -15,14 +16,41 @@
 
 import javax.xml.namespace.QName;
 import java.math.BigDecimal;
-import java.text.DecimalFormat;
-import java.util.*;
 import java.util.List;
+import java.util.*;
 
 /**
  * Created by Bryn on 12/29/2016.
  */
 public class LibraryBuilder {
+    public static final String SYSTEM = "System";
+    public static final String INTEGER = "Integer";
+    public static final String DECIMAL = "Decimal";
+    public static final String $_THIS = "$this";
+    public static final String HIGH = "high";
+    public static final String HIGH_CLOSED = "highClosed";
+    public static final String COMPATIBILITY_LEVEL_1_5 = "1.5";
+    public static final String $_INDEX = "$index";
+    public static final String $_TOTAL = "$total";
```

Comment:
Although I'm not opposed to the change here, it's not part of implementing the change for match resolution and I would prefer to see it in a different PR. Can we back this change out so we're not mixing changes?